dev/compose.yaml

```
@@ -83,22 +85,28 @@ services:
       start_period: 10s
 
   nextcloud:
-    image: nextcloud:${NEXTCLOUD_IMAGE_TAG:-apache}
-    restart: always
+    image: ghcr.io/juliusknorr/nextcloud-dev-php${NEXTCLOUD_IMAGE_PHP:-83}:${NEXTCLOUD_IMAGE_TAG:-master}
+    restart: unless-stopped
     entrypoint: bash
     command: /nextcloud.sh
     environment:
-      OVERWRITEPROTOCOL: 'https'
-      OVERWRITECLIURL: 'https://nextcloud.local'
-      PHP_UPLOAD_SIZE: '1024M'
-      POSTGRES_HOST: nextcloud-db
-      POSTGRES_DB: nextcloud
-      POSTGRES_USER: nextcloud
-      POSTGRES_PASSWORD: nextcloud
+      SQL: pgsql
+      VIRTUAL_HOST: nextcloud
+      SERVER_BRANCH: ${NEXTCLOUD_BRANCH:-stable31}
+      NEXTCLOUD_AUTOINSTALL: YES
+      NEXTCLOUD_TRUSTED_DOMAINS: nextcloud.local
```

```suggestion
      SQL: pgsql
      VIRTUAL_HOST: nextcloud
      SERVER_BRANCH: ${NEXTCLOUD_BRANCH:-stable31}
      NEXTCLOUD_AUTOINSTALL: YES
      NEXTCLOUD_TRUSTED_DOMAINS: nextcloud.local
      NEXTCLOUD_AUTOINSTALL_APPS:
```

Can we add this in the env? this will automatically enable the apps 
NEXTCLOUD_AUTOINSTALL_APPS